src/node_code_integrity.cc

```diff
@@ -0,0 +1,237 @@
+#include "node_code_integrity.h"
```

Comment:
This file should be surrounded by a ifdef of is windows. We don't need to compile this file for non-windows environments.